controllers/admin/AdminModulesController.php

```diff
@@ -690,24 +651,7 @@ public function postProcessCallback()
                 return;
             }
 
-            if ($key == 'check') {
```

Comment:
checkAndUpdate OK

but why disable 'check' ?

Reply:
because the check is only done when there is a checkAndUpdate, so dead code too.